script.js

```diff
@@ -0,0 +1,32 @@
+const words = [
+    APPLE,
+    ORANGE
+]
+
+//Save and transform to upper case the letter the user wrote:
+function tryLetter(){
+    let letter = document.getElementById('letter').value
+    let LETTER = letter.toUpperCase()
+}
+
+//Write the letter if correct, mark as a mistake if wrong:
+for(let i=0; i<words.length; i++){
+    if (LETTER === words[1][i]) {
+        return LETTER
+    } else {
+        if (document.getElementById('circle1').style.color === whitesmoke){
+            document.getElementById('circle1').style.color = purple
+        } else if(document.getElementById('circle2').style.color === whitesmoke){
+            document.getElementById('circle2').style.color = purple
+        }else if(document.getElementById('circle3').style.color === whitesmoke){
+            document.getElementById('circle3').style.color = purple
+        }else if(document.getElementById('circle4').style.color === whitesmoke){
+            document.getElementById('circle4').style.color = purple
+        }else if(document.getElementById('circle5').style.color === whitesmoke){
+            document.getElementById('circle5').style.color = purple
+        }else if (document.getElementById('circle5').style.color === purple){
+            document.getElementById('lostImg').style.display = block
+            document.getElementById('lostP').style.display=block
```

Comment:
And here also, what would you like this if/else statement to do exactly? And at what moment?

Reply:
Here is for the mistakes...(it won't be a person, like the hangman) if you make a mistake, the first circle turns purple, when you make the second mistake, the second circle turns purple...
when all the circles are purple, you are on your last chance...if you make a mistake, it will appear the image and the paragraph(you lost!)

Reply:
hmmm I'm thinking here that you will need some sort of counter inside the for loop that is checking for the matching letters
So maybe you have a counter for errors
So everytime you are checking for a new letter that the person has typed in, if there is an error (meaning, no match with any letters) the error counter goes up 1
And then using the information from the error counter, you can apply some styling to the circles
This counter will also help you determine when the person has lost, by comparing the number of errors with the amount of attempts the person can do 💡